### DISCUSSION TITLE
findUnique and mode insensitive

### DISCUSSION THREAD

**Discussion by alucardu** [prisma/prisma#9787] [Q&A]
I'm trying to return a user based on a email address but I would like to ignore cases, so no use can use the same email address/user name but with different capitalization:

```ts
const theUser = await prisma.user.findUnique({
  where: {
    email: String(args.email),
    mode: 'insensitive
  },
});
```
Returns error:

> Argument where of type UserWhereUniqueInput needs exactly one argument, but you provided email and mode.

On a `findyMany` it works:
```ts
    const theUser = await prisma.user.findMany({
      where: {
        email: {
          contains: String(args.email),
          mode: 'insensitive',
        },
      },
    });
```
But since this is regarding a single user object I feel that doing it this way is a bit off.

**Comment by ryands17** [comment]
@alucardu 👋 

You can use `findFirst`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
